aquarius/app/cached_block.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import logging
+import os
+from typing import Optional
+
+from flask_caching import Cache
+from aquarius.myapp import app
+
+logger = logging.getLogger(__name__)
+
+cache = Cache(
+    app,
+    config={
+        "CACHE_TYPE": "redis",
+        "CACHE_KEY_PREFIX": "ocean_aquarius",
+        "CACHE_REDIS_URL": os.getenv("REDIS_CONNECTION"),
+    },
+)
+
+
+def get_cached_block():
+    """
+    Retrieves the latest block value.
+    :return: `cached_block` stored in the database
+    """
+    return cache.get("cached_block")
+
+
+def update_cached_block(cached_block: Optional[int]):
```

Comment:
or, instead of making it Optional, make it a required int. Then you can remove the None check.